code/modules/reagents/machinery/reagentgrinder.dm

```diff
@@ -1,5 +1,21 @@
-/obj/machinery/reagentgrinder
 
+
+var/global/list/juice_items = list (
```

Comment:
The reason is in bees_apiary.dm

I wanted bees to process some fruit juice into their honey, but the seed datum only contains the NUTRIMENT chemical, so I needed the juice_items list to convert those nutriment into the corresponding juices.